website/pages/playground/index.html

```diff
@@ -41,291 +41,7 @@
-    }
-
-    .bottom-bar-buttons-right {
-      float: right;
-    }
-
-    .bottom-bar-buttons > * + * {
-      margin-left: 9px;
-    }
-
-    @media (max-width: 799px) {
-      .bottom-bar-buttons {
-        top: auto;
-        bottom: 100%;
-        padding: 2px;
-      }
-
-      .bottom-bar-buttons > * + * {
-        margin-left: 2px;
-      }
-    }
-
-    .options-container {
-      display: none;
-      box-sizing: border-box;
-      width: 225px;
-      height: 100%;
-      border-bottom: 1px solid #ddd;
-      border-right: 1px solid #ddd;
-      flex: 0 1 auto;
-    }
-
-    .options-container.open {
-      display: block;
-    }
-
-    .options {
-      display: flex;
-      flex-direction: column;
-      max-height: 100%;
-      overflow: auto;
-    }
-
-    .sub-options {
-      flex: 1;
-      display: flex;
-      padding: 15px 0 10px;
-      border-bottom: 1px solid #ddd;
-    }
-
-    .sub-options:last-child {
-      border: 0;
-    }
-
-    .sub-options > summary {
-      font-size: 14px;
-      font-weight: bold;
-      padding-bottom: 5px;
-      cursor: pointer;
-    }
-
-    .sub-options > summary:focus {
-      outline: 0;
-    }
-
-    .sub-options > * {
-      margin-left: 10px;
-    }
-
-    label {
-      font-family: Consolas, Courier New, Courier, monospace;
-      margin: 10px 0;
-      display: block;
-    }
-
-    input[type="number"] {
-      max-width: 40px;
-    }
-
-    /* Copied from the GitHub button styling. */
-    .btn {
-      box-sizing: content-box;
-      display: inline-block;
-      height: 18px;
-      padding: 0 5px;
-      border: 1px solid #d1d2d3;
-      border-radius: 0.25em;
-      background-image: linear-gradient(to bottom, #fafbfc, #e4ebf0);
-      font-size: 11px;
-      line-height: 18px;
-      font-weight: 600;
-      font-family: inherit;
-      color: #24292e;
-      text-decoration: none;
-      cursor: pointer;
-      outline: none;
-      position: relative;
-    }
-
-    .btn:focus {
-      border-color: #c8e1ff;
-    }
-
-    .btn:hover {
-      background-color: #e6ebf1;
-      background-image: linear-gradient(to bottom, #f0f3f6, #dce3ec);
-      border-color: #afb1b2;
-    }
-
-    .btn:active {
-      background-color: #e9ecef;
-      background-image: none;
-      border-color: #afb1b2;
-      box-shadow: inset 0 0.15em 0.3em rgba(27, 31, 35, 0.15);
-    }
-
-    .tooltip {
-      position: absolute;
-      z-index: 1;
-      bottom: 100%;
-      left: 50%;
-      transform: translateX(-50%);
-      margin-top: 4px;
-      padding: 0.4em 0.8em;
-      background-color: #000;
-      color: #fff;
-      border-radius: 0.4em;
-      pointer-events: none;
-    }
-
-    .tooltip::before {
-      content: "";
-      position: absolute;
-      top: 100%;
-      left: 50%;
-      transform: translateX(-50%);
-      border: 6px solid transparent;
-      border-bottom: none;
-      border-top-color: #000;
-    }
-  </style>
+  <link rel="stylesheet" href="/playground.css">
```

Comment:
It seems you forgot to apply the `separateCss`, and the file should be in a folder since it requires a folder:

> `separateCss`: Folders inside which any css files will not be processed and concatenated to Docusaurus's styles.

![image](https://user-images.githubusercontent.com/8341033/34386614-7ac0f6bc-eb64-11e7-8d0d-271af846fb6a.png)

Reply:
Oh, I am sorry, forgot to add `separateCss` property. 
It also works with files, and if there will be more than one file we can move them to the folder.